src/dash-adapter.js

```diff
@@ -904,7 +915,7 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
    */
   selectTextTrack(textTrack: TextTrack): void {
     if (this._shaka && textTrack instanceof TextTrack && !textTrack.active && (textTrack.kind === 'subtitles' || textTrack.kind === 'captions')) {
-      this._shaka.setTextTrackVisibility(this._config.textTrackVisibile);
+      this._shaka.setTextTrackVisibility(this._config.useShakaTextTrackDisplay || this._config.textTrackVisibile);
```

Comment:
we need this because we set useNativetextTrack?

Reply:
No, because we want shaka will render it

Reply:
so let's reuse the flag for useNativeTextTrack as discussed earlier.